File: pythonTestPractice/Calculator.java
public class Calculator {
    public static int palindrome(int num){
        int eachDigit = 0;
        int copy = num;
        int reverse = 0;
        while (num > 0){
        
            eachDigit = num % 10;
            reverse = (reverse * 10) + eachDigit;
            num /= 10;   
        }
        
        if (copy == reverse){
            return reverse;
        }else{
            return 0;        
        } 

 
        
    }
}

File: pythonTestPractice/calculator.py
def add(num1, num2):
    return num1 + num2
    
def palindrome(num):

    copy = num
    reverse = 0
    while num > 0:
        eachdigit = num % 10
        reverse = (reverse * 10) + eachdigit
        num //= 10
    
    if copy == reverse:
        return reverse
    else: 
        return 0
    
    
